### DISCUSSION TITLE
New KICS Logo

### DISCUSSION THREAD

**Discussion by cx-antero-silva** [Checkmarx/kics#5835] [General]
Hi Community,

We are considering changing Kics Logo and we came up with some examples. We would love your opinion on them. 

Please review the document in attach and give us some feedback of the ones you like the most. Ideas for new logos will also be appreciated. 

Thank you in advance.

Regards,
Antero Silva
(Kics Product Manager)

[Checkmarx KICs Logo Concepts.pdf](https://github.com/Checkmarx/kics/files/9636321/Checkmarx.KICs.Logo.Concepts.pdf)

**Comment by cx-ori-bendet** [comment]
2 and 3 are my favorites
